.circleci/config.yml

```diff
@@ -17,6 +17,12 @@ jobs:
           command: |
             bundle install --jobs=4 --retry=3 --path vendor/bundle
 
+      # run rubocop checks
```

Comment:
Idea: drop all the comments - you already have VERY good `name` properties.

Reply:
you're right, those comments are redundant